TimeTrackerManager/TimeTrackerManager/Core/Controllers/Clients/ClientsTableViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ProjectsTableViewController.swift
+//  TimeTrackerManager
+//
+//  Created by Bocanu Mihai on 25.01.2022.
+//
+
+import UIKit
+import TimeTrackerCore
+
+class ClientsTableViewController: UITableViewController {
```

Comment:
Trebuie facut un refresh la tabel dupa ce adaugi client nou. 
Si baga-i si un `Pull to Refresh`. Baga si pe users